contentcuration/contentcuration/frontend/channelEdit/views/StudioTree.vue

```diff
@@ -151,4 +159,10 @@
 .node-item {
   cursor: pointer;
 }
+
+.slide-y-transition-enter-active,
+.slide-y-transition-leave-active {
+  transition-duration: .2s
+}
```

Comment:
FYI, one thing we'll be standardizing in the design system is transitions. We'll be doing something like this:

https://learningequality.slack.com/archives/CB37UM23A/p1582842552026100?thread_ts=1582835466.022500&cid=CB37UM23A

Reply:
Ah thanks for flagging! I'll update the speed here for now